src/asset_folder_importer/asset_folder_sweeper/find_files.py

```diff
@@ -0,0 +1,71 @@
+import os
+from pprint import pprint
+import logging
+import mimetypes
+from asset_folder_importer.asset_folder_sweeper.posix_get_mime import posix_get_mime
+import re
+
+def find_files(cfg,db):
+    #Step three. Find all relevant files and bung 'em in the database
+    startpath = cfg.value('start_path',noraise=False)
+
+    print "Running from '%s'" % startpath
```

Comment:
this should be replaced with a call to Logger.  See other assetsweeper code for examples, but I'd recommend something like:

```logger = logging.getLogger(__name__)```

at line 7, then you can replace a print call like this with:

```logger.info("Running from '%s'" % startpath)```